docs/wiki/deployment/configuration.md

```diff
@@ -188,56 +205,197 @@ And the same configuration file from the OS X example is appropriate.
+many services and programs, while ensuring that only relevant queries will be
+executing on your host.
+
+You don't need to define any discovery queries for a pack. If no discovery
+queries are defined, then the pack will always execute.
+
+Discovery queries look like:
 
 ```json
 {
-  "queries": {
-    "active_directory": {
-      "query": "select * from ad_config;",
-      "interval": "1200",
-      "platform": "darwin",
-      "description": "Check each user's active directory cached settings."
-    },
-    "full_disk_encryption": {
-      "query": "select * from disk_encryption;",
-      "interval": "86400",
-      "description": "Monitor for newly-encrypted/unencrypted disks."
-    }
-  }
+	"discovery": [
+		"select pid from processes where name = 'foobar';",
+		"select pid from processes where name = 'baz';"
+	],
+	"queries": {}
 }
 ```
 
-A query pack may make wider limitations about how the queries apply too:
+In the above example, the pack will only execute on hosts which are running
+processes called `foobar` and `baz`.
+
+Discovery queries are refreshed for all packs every 60 minutes. You can
+change this value via the `pack_refresh_interval` configuration option.
+
+### Where do packs go?
+
+The default way to define a query pack is in the main configuration file.
+Consider the following example:
 
 ```json
 {
-  "queries": {...},
-  "platform": "ubuntu",
-  "version": "1.4.5"
+	"options": {
+		"enable_monitor": "true"
+	},
+	"packs": {
+		"foo": {
+			"queries": {}
+		},
+		"bar": {
+			"queries": {}
+		}
+	}
 }
 ```
 
-Then every query within will only be added to a schedule if the osqueryd process is running on a Ubuntu distro with a minimum osquery version of 1.4.5.
+Alternatively, however, you can also define the value of a pack as a raw
+string. Consider the following example:
+
+```json
+{
+	"options": {
+		"enable_monitor": "true"
+	},
+	"packs": {
+		"foo": "/tmp/foo.json",
+		"bar": "/tmp/bar.json"
+	}
+}
+```
+
+In the above example, the packs are defined using a local filesystem path.
+When osquery notices that you've defined the value of your pack via a string
+rather than an inline dictionary, the active config plugin is called to resolve
+what should be done to go from `/tmp/foo.json` to the actual content of the
+pack.
+
+To accomplish this, the config plugin in question must implement the virtual
+method `Status genPacks(const std::string& name, const std::string& value,
+std::string& pack);`, which is defined by the base `ConfigPlugin` class.
+
+If you're using the filesystem config plugin, which is the default option, this
+method is defined already and will read string paths from the filesystem. If
+you use a custom config plugin and would like to use the packs shorthand
+syntax, then be sure to implement the `genPacks` method in your config plugin.
+
+### Options
+
+In addition to `discovery` and `queries`, a pack may contain a `platform` key
```

Comment:
I would recommend not using back-ticks unless you're referring to code or calling out a path or command *for first time*. The RTD tick styling is obnoxious and distracting. Unfortunately, too many ticks makes a page look confusing.